website/content/en/status/report-2023-07-2023-09/openssl3.adoc

```diff
@@ -1,4 +1,4 @@
-=== OpenSSL 3 in base - improved
+=== OpenSSL 3 in base -- improved
```

Comment:
Uppercase I for Improved? If there'll be Americanisation of headings.

Reply:
I would go with uppercase I.